integration/fixture_test.go

```diff
@@ -80,11 +134,14 @@ func (f *fixture) testDirPath(s string) string {
 }
 
 func (f *fixture) installTilt() {
+	f.t.Helper()
```

Comment:
TIL

there are so many places the lack of this has annoyed me!

Reply:
You can see where I've broken a test based on this getting added 😶